bin/rigger-install-terraform

```diff
@@ -5,15 +5,10 @@ set -eo pipefail -o nounset
 function install-terraform {
   local os="$(uname | tr '[:upper:]' '[:lower:]')"
   local arch="$(uname -m)"
-  local terraform_version="0.6.3"
+  local terraform_version="0.6.4"
   local terraform_base_url="https://dl.bintray.com/mitchellh/terraform"
   local terraform_filename
 
-  if command -v terraform &>/dev/null; then
-    rigger-log "Terraform already available ($(command -v terraform))."
-    return 0
-  fi
-
```

Comment:
This is critical due to the recently found bug in Terraform with DigitalOcean (affects 0.6.5-0.6.6)